test/test_files/copy/copy_long_string.test

```diff
@@ -1,20 +1,13 @@
 -GROUP CopyLongStringTest
--DATASET CSV copy-fault-tests/long-string
--SKIP
+-DATASET CSV tinysnb
 
 --
 
 -CASE LongStringError
--DEFINE AliceStr REPEAT 819 "Alice"
 
--LOG LongStringAlice
--STATEMENT MATCH (p:person) RETURN p.fName;
----- 2
-${AliceStr}A
-Bob
-
--LOG Gender
--STATEMENT MATCH (p:person) RETURN p.gender
----- 2
-1
-2
+-LOG LongStringError
```

Comment:
Why is this file in a different place?

Reply:
This test should throw an error, they are different tests.